client/src/components/DataFiles/DataFilesModals/DataFilesCopyModal.js

```diff
@@ -124,9 +125,9 @@ const DataFilesCopyModal = React.memo(() => {
     [selectedFiles]
   );
 
-  const actionString = `Copying ${selected.length} File${
-    selected.length > 1 ? 's' : ''
-  }`;
+  const actionString = `${
+    params.system === modalParams.system ? 'Copying' : 'Start Copying'
```

Comment:
Have we run this wording by CMD? These seem to convey the same meaning to me.

Reply:
This is what I got from Wes, in which he just mentioned making the case consistent.
https://tacc-team.slack.com/archives/CQUA4D5KJ/p1633444635075600?thread_ts=1633386166.074700&cid=CQUA4D5KJ

Should I just revert this part if there isn't much difference?